README.md

```diff
@@ -49,16 +49,26 @@ snowdrop	    https://snowdrop.github.io/helm
 ## Usage
 
 Requirements:
-- Connected/logged to a Openshift cluster
+- Connected/logged to a Kubernetes/Openshift cluster
 - Have installed [the Helm command line](https://helm.sh/docs/intro/install/)
 
 ### spring-boot-example-app
 
-This chart deploys and exposes a Spring Boot application in Openshift.
-To install it, you need to execute the following command:
+This chart deploys and exposes a Spring Boot application in Kubernetes or Openshift.
+
+- For Kubernetes:
+To use it in Kubernetes, the image of the Spring Boot application needs to be available in a public container registry. In this example, we'll use the image `quay.io/user/my-app:latest`. To install it, you need to execute the following command:
 
 ```console
-$ helm install my-spring-boot-app snowdrop/spring-boot-example-app --set name=app --set s2i.source.repo=http://github.com/org/repo --set s2i.source.ref=main
+$ helm install my-spring-boot-app snowdrop/spring-boot-example-app --set name=app --set docker.image=quay.io/user/my-app:latest --set ingress.host=<your kubernetes domain>
+```
+
+**note**: if you want to expose your application in Kubernetes, you need to provide the `ingress.host` property.
+
+- For Openshift, to install it, you need to execute the following command:
```

Comment:
Take care about the OpenShift name please

Reply:
changed